guides/v2.1/coding-standards/technical-guidelines.md

```diff
@@ -193,7 +193,7 @@ class Config
     </tr>
     <tr>
         <td>
-{% highlight php %}
+```php
```

Comment:
Will the code gates render correctly within an HTML table?

Reply:
It seems to with proper spacing/indentation. I will check all the PHPs.